### PR TITLE
Use class constants inside IO bundle tests

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

**Comment by andrerom** [comment]
But tests are failing.


would probably be easier if you bundle several in one PR as long as it is still reviewable.

**Comment by MarioBlazek** [comment]
@andrerom this build was affected by Packagist problem. Can you please restart build ?

**Comment by MarioBlazek** [comment]
@andrerom build restarted, all tests passing.

File: phpunit.xml [hunk 1]
```diff
@@ -99,3 +99,4 @@
     </whitelist>
   </filter>
 </phpunit>
+
```
Comment (andrerom) [inline phpunit.xml]: niptick, but seems this was added by mistake?

**Comment by MarioBlazek** [comment]
@andrerom whitespace removed.

**Pull request merged**